### DISCUSSION TITLE
Do I need to install Privacy Badger (and ClearURLs) as my privacy blocker when having UBlock Origin Lite?

### DISCUSSION THREAD

**Discussion by thedoggybrad** [uBlockOrigin/uBOL-home#152] [Q&A]
Thanks to the one who responded in my thread regarding the custom filters which are unnecessary with UBlock Origin Lite and it is true so I finally migrated all my managed systems to UBlock Origin Lite to prepare for Manifest v2 depreciation. 

But now, with EasyPrivacy and other anti-tracking custom filters I use become unavailable to UBlock Origin Lite, is it justifiable or recommended to install Privacy Badger to replace that or it will interfere with UBlock Origin Lite?

And also the ClearURLs extension or its userscript alternative (I use the Unofficial ClearURLs filterlist for UBO before) or is it built in to the lite version?

So, all in all, is it okay to install ClearURLs and Privacy Badger to replace the functions of the UBlock Origin (MV2) achieved by custom filterlists?

**Comment by garry-ut99** [comment]
> Privacy Badger, But now, with EasyPrivacy and other anti-tracking custom filters I use become unavailable to UBlock Origin Lite

Flawed assumption, EasyPrivacy isn't displayed in uBlock Lite, unlike EasyList which is displayed, but despite it, uBLock Lite still does contain EasyPrivacy, it's embeded in "Ads, trackers, miners, and more".

Also, this issue explains how to check which filter lists are included in uBlock Lite:
- https://github.com/uBlockOrigin/uBOL-home/issues/149

As seen in this comment: https://github.com/uBlockOrigin/uBOL-home/issues/149#issuecomment-2222799157 , EasyPrivacy is contained in uBlock Lite
`Fetched local https://ublockorigin.github.io/uAssets/thirdparties/easyprivacy.txt `

> ClearURLs

There exists "AdGuard URL Tracking Protection" ([LINK](https://adguard.com/kb/general/ad-filtering/adguard-filters/)) in uBlock Lite.

**Comment by thedoggybrad** [comment]
Did not realized that, I am basically overloading my web browser with unnecessary and redundant web browser extensions.
